composer.json

```diff
@@ -16,13 +16,13 @@
     "homepage":"https://github.com/aloha/laravel-twilio",
     "license": "MIT",
     "require": {
-        "php": ">=7.2.0",
+        "php": ">=7.2.0|^8.2",
```

Comment:
I would create a new `aloha/twilio` version `6.0.0` where only PHP 8.2 is supported, and Laravel 10 and 11.